### ISSUE TITLE
"Star" links under "Fork us on Github" only link to project repo

### ISSUE THREAD

**Issue by non-Jedi** [vector-im/vector.im#18]
For example "Riot Web" links to https://github.com/vector-im/riot-web, and the "Star" link next to it also links to https://github.com/vector-im/riot-web.

**Comment by nadonomy** [comment]
Hey @non-Jedi— did you mean to report this in this repo? https://github.com/vector-im/riot-web/

Closing this issue as this repo is for the vector.im site.

**Comment by non-Jedi** [comment]
Nope. This one, like all the others I reported is for the about.riot.im website. I put them here as feedback since it seemed the most relevant place.

**Comment by ara4n** [comment]
i am confused - what should it be linking to if not the GH repo?

**Comment by non-Jedi** [comment]
Not sure (and this is a niggle in any case). It's just a bit weird to see the link to the repo and right next to it a link that says "star" while looking like a button instead of a link. It's the same style of button used in the github ui for the button that actually stars a repo.

![p1](https://user-images.githubusercontent.com/17915621/54779965-f8347d80-4bee-11e9-8614-f74896bb4946.png)

Clicking it, I expected it to somehow actually star the repo in question on github (which probably isn't possible, but I'm not a web guy, so I don't know), and instead it just took me to the repo. Maybe instead of "[⭐ star] 2,635" you could do "⭐ 2,635 stars" or something. I guess the big point is that buttons styled like that generally *do* something on webpages rather than just being links.

In any case, this is all super trivial and nit-picky, so sorry about that.

**Comment by tredondo** [comment]
I also wanted to report the same thing. A "Stars" link just seems pointless.

It should be a badge with the #. Or maybe remove it?
